test/symbol-store-tests.js

```diff
@@ -27,60 +27,97 @@ const SymbolStore = require('../lib/symbol-store').SymbolStore;
 
 chai.should();
 
-describe('Basic examples', function () {
+describe('SymbolStore tests', function () {
     it('Empty', function () {
         const store = new SymbolStore();
-        store.ListSymbols().length.should.equal(0);
-        store.ListTranslations().length.should.equal(0);
+        store.listSymbols().length.should.equal(0);
+        store.listTranslations().length.should.equal(0);
     });
 
     it('One item', function () {
```

Comment:
I would like that the tests names would flow with the sentence `It ...` as most of the other tests already do, but this is a super minor nitpick